src/serializers/GltfSerializer.cpp

```diff
@@ -121,17 +121,28 @@ const uint32_t component_type<int>::value = CT_UNSIGNED_INT;
 template <>
 const uint32_t component_type<float>::value = CT_FLOAT;
 
+static int bufferViewId = 0;
```

Comment:
For this we need to find another solution though. It's possible in Python to have multiple gltf serializers open at once. They can't use the same static counter. It seems every call to write_accessor() is going to increment bufferViewid with 1 so I would recommend moving the counter to a local variable at the callsite of write_accessor and passing it as a function parameter (didn't check if possible, but something like that).

Reply:
ok sure. I have changed bufferViewId into a member variable and I now send it as a parameter to write_accessor.